WcaOnRails/app/models/competition.rb

```diff
@@ -68,6 +68,7 @@ class Competition < ApplicationRecord
       ).group(:id)
   }
   scope :order_by_date, -> { order(:start_date, :end_date) }
+  scope :order_by_announcement_date, -> { order(:announced_at) }
```

Comment:
I definitely agree we want to show the latest announced first!
Since this scope is basically used only in that context, I would order them appropriately right away:
```suggestion
  scope :order_by_announcement_date, -> { order(announced_at: :desc) }
```